Makefile

```diff
@@ -30,8 +30,10 @@ doc:
 get:
 	$(GOGET) -t -v ./...
 
-test: get
+fmt:
 	$(GOFMT) ./...
+
+test: get fmt
```

Comment:
@filipecosta90 you normally don't run fmt automatically as part of the test/unittest phase as it's not expected